gulp/tasks/test_debug.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        paths = {
+            spec: [
+                'test/specs/**/*.js'
```

Comment:
Good question. I'd have to use a similar trick to the debug config, where both files load a file in a relative directory which sets something into the global namespace. My thinking was that there is very little to this configuration and it shouldn't need to change, but I agree that it would be better if it was shared. Let me see what I can come up with.